packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -59,23 +59,11 @@ const withPathPrefix = (url, pathPrefix) =>
   (pathPrefix + url).replace(/\/\//, `/`)
 
 // TODO: remove this check with next major release
-// the getCache feature was implemented in gatsby@2.0.88
-const safeGetCache = ({ getCache, cache, reporter }) => {
-  const GATSBY_VERSION_WITH_GET_CACHE = `2.0.88`
-  let warned = false
-  return id => {
-    if (!getCache) {
-      if (!warned) {
-        warned = true
-        reporter.warn(reporter.stripIndent`
-          It looks like you're using a slightly outdated version of gatsby with gatsby-transformer-remark.
-          To update, run 'npm install gatsby@^${GATSBY_VERSION_WITH_GET_CACHE} --save' or 'yarn add gatsby@^${GATSBY_VERSION_WITH_GET_CACHE}'
-        `)
-      }
-      return cache
-    }
-    return getCache(id)
+const safeGetCache = ({ getCache, cache }) => id => {
```

Comment:
Note for posterity, we _already_ warn on conflicting peerDeps, so a second warning would be sorta redundant.